docs/paper/admin/reference/system-properties.md

```diff
@@ -0,0 +1,183 @@
+---
+slug: /reference/system-properties
+---
+
+# Paper System Properties 
+
+These system properties can be set when you start your server allowing for the configuration of various settings.
+
+:::danger Danger Ahead
+
+Setting flags for the JVM can alter how it operates and the same goes for the paper server.
+If you are unsure about what a flag does, it is recommended that you **do not use it**.
+
+:::
+
+## How they work
+
+System properties are set when you start your server. For example, if you are using a `.bat` or a `.sh` file to start your server, you can add the system properties to the file. For example:
+
+```bash
+java -Dpaper.log-level=FINE -jar paper.jar
+```
+
+Where a `-D` is used to set a system property, and the system property is `paper.log-level` with a value of `FINE`. Otherwise, just add them to the start command
+
+## List of System Properties
+
+#### paper.playerconnection.keepalive:
+
+- **default**: `30`
+- **description**: Controls how long the player connection will wait before closing when not receiving any keepalives, in seconds.
+
+#### paper.ticklist-excessive-delay-threshold:
+
+- **default**: `60`
+- **description**: Sets the excessive tick delay to the specified number of seconds.
```

Comment:
This doesn't really explain what excessive ticks mean. Would be nice to further explain here.

Reply:
Seems like this flag has been removed :P - I might just remove this from the docs